scylla/src/routing/partitioner.rs

```diff
@@ -37,6 +37,7 @@ impl PartitionerName {
     }
 }
 
+impl sealed::Sealed for PartitionerName {}
 impl Partitioner for PartitionerName {
```

Comment:
:wrench: commit message mistake: `We will un-pub them in the next commit.` -> should be without `un-`.